commands/service_deploy_test.go

```diff
@@ -13,3 +18,37 @@ func TestServiceDeployCmdFlags(t *testing.T) {
 	flags.Set("env", "a=1,b=2")
 	require.Equal(t, map[string]string{"a": "1", "b": "2"}, c.env)
 }
+
+func TestServiceDeploy(t *testing.T) {
```

Comment:
All tests of command inside one test:
- env
- statuses
- output

Should be spleted